src/HomeKit/HMMutablePresenceEvent.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using XamCore.ObjCRuntime;
+
+namespace XamCore.HomeKit {
+
+	partial class HMMutablePresenceEvent {
+
+		public HMPresenceType PresenceType {
```

Comment:
Mmh sure but I don't follow the logic. Why is virtual needed? Wound't we just remove the manual bindings?